calico-enterprise/operations/cnx/configure-identity-provider.mdx

```diff
@@ -42,215 +43,242 @@ When configuring your cluster, you may be asked for the following inputs:
-
-   ```yaml
-   apiVersion: operator.tigera.io/v1
-   kind: Authentication
-   metadata:
-     name: tigera-secure
-   spec:
-     managerDomain: https://<domain-of-manager-ui>
-     oidc:
-       issuerURL: <your-idp-issuer>
-       usernameClaim: email
-   ```
-
-1. (Optional) Google OIDC does not support the groups claim. However, {{prodname}} leverages [Dex IdP](https://dexidp.io/docs/connectors/google/) to add groups if you configure a service account.
-   This account needs Domain-Wide Delegation and permission to access the `https://www.googleapis.com/auth/admin.directory.group.readonly` API scope.
-   To get group fetching set up:
-
-   - Follow the [instructions](https://developers.google.com/admin-sdk/directory/v1/guides/delegation) to set up a service account with Domain-Wide Delegation
-     - During service account creation, a JSON key file will be created that contains authentication information for the service account.
-     - When delegating the API scopes to the service account, delegate the `https://www.googleapis.com/auth/admin.directory.group.readonly` scope and **only this scope**.
-   - Enable the [Admin SDK](https://console.developers.google.com/apis/library/admin.googleapis.com/)
-   - Use the `serviceAccountSecret` and `adminEmail` configuration options in the next step.
-     - The contents of the JSON key file should be used as the `serviceAccountSecret`
-     - For `adminEmail` choose a G Suite super user. The service account will impersonate this user when making calls to the admin API.
-
-1. Apply the secret to your cluster with your OIDC credentials.
-
-   ```yaml
-   apiVersion: v1
-   kind: Secret
-   metadata:
-     name: tigera-oidc-credentials
-     namespace: tigera-operator
-   data:
-     clientID: <your-base64-clientid>
-     clientSecret: <your-base64-clientid-secret>
-       # If you created a service account in the previous step, include the following two fields.
-     serviceAccountSecret: <your-base64-json-contents>
-     adminEmail: <your-base64-gsuite-user>
-   ```
-
-</TabItem>
-<TabItem label="OpenShift" value="OpenShift-2">
-
-1. Install the [Openshift CLI (oc)](https://docs.okd.io/4.9/cli_reference/openshift_cli/getting-started-cli.html).
-
-1. Create values for some required variables. `MANAGER_URL` is the URL where {{prodname}} Manager will be accessed,
-   `CLUSTER_DOMAIN` is the domain (excl. port) where your OpenShift cluster is accessed (run `oc status` to get it) and `CLIENT_SECRET` is a value of your choosing.
-
-   ```bash
+  <TabItem label="OIDC" value="OIDC-0">
+
+    1. Apply the Authentication CR to your cluster to let the operator configure your login.
+    The following example uses the email claim. The email field from the JWT (created by your IdP), is used as the username for binding privileges.
+
+    ```yaml
+    apiVersion: operator.tigera.io/v1
```

Comment:
This yaml and all others have lost their tabs.

Reply:
Fixed